src/ja/server/diagrams/finishJob.puml

```diff
@@ -0,0 +1,43 @@
+@startuml
```

Comment:
This diagram is misguided.
Here is what should happen when a Job finishes:
* Server gets information about Job being finished
* Server re-evaluates which Jobs to run on which WorkerClients. This is the **exact same** process like when a Job is added by a user.